smartdevicelink/sdl_java_suite
# Update urls to use https instead of http

Fixes #1333 

This PR is **[ready]** for review.

### Risk
This PR makes **[no]** API changes.

### Summary
replace http with https where needed

### Test Steps
- Add logging to SDLProxyBase and PolicyFetcher to observe URLs
- Run Android Test App and connect to manticore
- Update the policy table url in manticore
- Observe Logs to ensure Http is being replaced with Https in URLs and the app is able to communicate with the policy table

### CLA
